src/components/Preview.jsx

```diff
@@ -13,7 +13,7 @@ class Preview extends React.Component {
     bindAll(this, '_handlePopOutClick');
   }
 
-  _generateDocument() {
+  _generateDocument(isPopout = false) {
```

Comment:
WAIT I’m a moron.

We should actually be taking this param and using it for _all_ the options being passed to `generatePreview` below.

Still stand by the idea of keeping the popout semantics confined to this component tho